core/chains/evm/logpoller/orm_test.go

```diff
@@ -28,7 +28,7 @@ func setup(t testing.TB) (*ORM, *ORM) {
 	return o1, o2
 }
 
-func TestORM_GetBlocks(t *testing.T) {
+func TestORM_GetBlocks_From_Range(t *testing.T) {
```

Comment:
I think we should add a test case with 256 blocks in the range, just to ensure that this very common case works for the new method.